packages/rocketchat-livechat/app/client/stylesheets/main.less

```diff
@@ -774,6 +928,8 @@ input:focus {
 	opacity: 0.5;
 	align-self: flex-end;
 	transition: opacity 0.15s ease-out;
+  	bottom: 0;
```

Comment:
@HenrikScholl what's the effect if this is not set?
Should/Could this also be limited to `@media @smartphones`

Reply:
This Code set a fix postion to the Chat Input-Form (Issue 88)